data.sql

```diff
@@ -11,3 +11,23 @@ VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (5, 'Charmander', '2020-02-08', 0, false, 11);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (6, 'Plantmon', '2021-11-15', 2, true, 5.7);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (7, 'Squirtle', '1993-05-02', 3, false, 12.13);
```

Comment:
- [ ] The animal "Squirtle" was born on "1993-04-02" but it seems you got the month wrong so please make sure to update the date here